### PR TITLE
upterm: 0.7.3 -> 0.8.2

### PR DESCRIPTION
**Pull request by felixsinger**
###### Description of changes

Update to latest version.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/upterm/default.nix [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   pname = "upterm";
-  version = "0.7.3";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "owenthereal";
     repo = "upterm";
     rev = "v${version}";
-    hash = "sha256-eEzFqKYhsG5e1DVLWJq08NM9xyfn1yPNV0NIgOErj4E=";
+    sha256 = "1yxj5c1gq6ph986zh1bgkmdwc56bdlxs7c9wifzfy7py7sr0bi95";
```
Comment (mweinelt) [inline pkgs/tools/misc/upterm/default.nix:15]: Let's not rollback from hash to `sha256` please.

Comment (felixsinger) [reply pkgs/tools/misc/upterm/default.nix:15]: How do I get the other format? I failed figuring this out.

Comment (mweinelt) [reply pkgs/tools/misc/upterm/default.nix:15]:
```
❯ nix hash to-sri --type sha256 1yxj5c1gq6ph986zh1bgkmdwc56bdlxs7c9wifzfy7py7sr0bi95
sha256-JcUFsj7+Hu++izyxozttyxTGW51vBfgNSvAa/AIrsvs=
```

**Review by hax404 — APPROVED** [review]
works

File: pkgs/tools/misc/upterm/default.nix (hunk 1 above)
Comment (Artturin) [reply pkgs/tools/misc/upterm/default.nix]: nix versions 2.4 and after give you sri hash by default fyi

**Pull request merged**